### PR TITLE
Bump `open-autonomy`

### PR DESCRIPTION
**Pull request by Adamantios**
## Proposed changes

Bump `open-autonomy` from `0.3.2` to `0.3.3`.

## Fixes

n/a

## Types of changes

What types of changes does your code introduce? (A **breaking change** is a fix or feature that would cause existing functionality and APIs to not work as expected.)
_Put an `x` in the box that applies_

- [ ] Non-breaking fix (non-breaking change which fixes an issue)
- [ ] Breaking fix (breaking change which fixes an issue)
- [ ] Non-breaking feature (non-breaking change which adds functionality)
- [ ] Breaking feature (breaking change which adds functionality)
- [x] Refactor (non-breaking change which changes implementation)
- [ ] Messy (mixture of the above - requires explanation!)

## Checklist

_Put an `x` in the boxes that apply._

- [x] I have read the [CONTRIBUTING](../blob/main/CONTRIBUTING.md) doc
- [x] I am making a pull request against the `main` branch (left side). Also you should start your branch off our `main`.
- [x] Lint and unit tests pass locally with my changes
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have locally run services that could be impacted and they do not present failures derived from my changes

## Further comments

n/a

### CODE REVIEW

**Review by DavidMinarsch — COMMENTED** [review]
For the release - please also add an entry in `History.md`

File: packages/packages.json [hunk 1]
```diff
@@ -1,22 +1,22 @@
...
-    "skill/valory/apy_estimation_abci/0.1.0": "bafybeibcwnmomikx6z5muoqykiif3rp7xmmxeqkbodkut5d745utcssjkm",
-    "skill/valory/registration_abci/0.1.0": "bafybeigck3gb5tvm5tilzii7kvzqpryxrmxu2meck5empgl7lhuz66mbgq",
-    "skill/valory/apy_estimation_chained_abci/0.1.0": "bafybeibbwkbgak2yyuqwzzwdub3chthnuy3cqfivhntlv2eymhvcfnrnwi",
-    "agent/valory/apy_estimation/0.1.0": "bafybeichwec4gfanvwm74lx6nfkmonzcmdmilekaytwbktwdqfjnz7tg5a",
+    "skill/valory/abstract_round_abci/0.1.0": "bafybeicwhcpuov2bvfwvzxdvmy6fhmijdhqi3uthee3bv4cfe7wm5fsbjy",
+    "skill/valory/apy_estimation_abci/0.1.0": "bafybeieqzekji2vwmro7pdqibcfe37porr55an2rjlz4x2rnjmbccneafu",
+    "skill/valory/registration_abci/0.1.0": "bafybeievtpfxmzrdueqgisebeqkefukvfhzhiczvmdnp4rerwup4v6karm",
+    "skill/valory/apy_estimation_chained_abci/0.1.0": "bafybeif4ihwjajiyinbbbqhaejpu7f2kkzf4xwtsy2hakaqyxwo2ig76am",
+    "agent/valory/apy_estimation/0.1.0": "bafybeicxafdcodmkbfzhikkkqhxhr55inxwhvwbjeakx4ywjp4hrhu5eo4",
     "contract/valory/offchain_aggregator/0.1.0": "bafybeibp2cbuaxj4sthoaf5vdbdos6am2kqcbomrt2x6cyqsmxu34jabbu",
```
Comment (Adamantios) [inline packages/packages.json:20]: Oh, I just noticed this. I think `offchain_aggregator` should not be here.

Comment (Adamantios) [reply packages/packages.json]: Removed in ba03d36257764c180484ba52733c00e414a17f2d.

**Review by 0xArdi — APPROVED** [review]

File: HISTORY.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
+# Release History - `apy-oracle`
+
+## 0.3.3 (2022-10-24)
+
+- Bumped open-autonomy to `0.3.3` #6
```
Comment (Adamantios) [inline HISTORY.md:5]: I wonder if this should be `0.1.0` instead.

Comment (DavidMinarsch) [reply HISTORY.md:5]: I was thinking the same, but then actually like the fact that you started at the height where we branched. Going forward, they can of course diverge, if needed.

**Review by DavidMinarsch — APPROVED** [review]

**Pull request merged**